src/Sulu/Bundle/AdminBundle/Resources/js/components/Radio/Radio.js

```diff
@@ -1,18 +1,22 @@
 // @flow
 import React from 'react';
 import type {ElementRef, Node} from 'react';
+import classNames from 'classnames';
 import radioStyles from './radio.scss';
 
 type Props = {
+    name: string,
     checked: boolean,
-    value: string,
-    onChange?: (value: string) => void,
-    children: Node,
+    value: string | number,
+    skin?: 'dark' | 'light',
```

Comment:
Can you clarify if we need both skins? And if yes, decide on which one is the default.